# SOG infile patch to produce a testcase infile from the repository infile

# This patch is used by buildbot.

# It was created with:
#    $ cd SOG
#    $ cat tests/infile_patches/patch_comments > path_to/patch_file
#    $ diff -u infile path_to/sample_new_infile >> path_to/patch_file

#  To apply this patch manually, do:
#    $ cd SOG
#    $ patch -i path_to/patch_file -o path_to/new_infile

# where:
#   - tests/infile_patches/patch_comments contains these comments
#   - path_to/patch_file is the path and name of this file
#   - path_to/new_infile is the path and the name of the new infile to be
#     created.

--- infile	2010-05-13 19:11:03.000000000 -0700
+++ infiles/R3base	2010-05-13 19:11:01.000000000 -0700
@@ -1,20 +1,19 @@
 ! -*- mode: f90 -*-  ! This line must be at the top of the file!
 
-! SOG code infile template
-
-! SOG code infile for 356.5 day run starting at cruise 04-14 station S3
-! CTD cast (2004-10-19 12:22 LST).
+! SOG code infile for ~1 yr run starting at cruise 03-20 station S3
+! CTD cast.  This is the R3 baseline regression test infile.  All
+! biology model complexity is turned off.
 
 ! Grid parameters:
 "maxdepth"	40.d0	"depth of modelled domain [m]"
 "gridsize"	80	"number of grid points"
-"lambda"	0.d0	"grid spacing parameter"
+"lambda"	 0.d0	"grid spacing parameter"
 
 ! Run timing parameters:
 ! Initial conditions date/time (must match a CTD profile date/time)
-"init datetime"	"2004-10-19 12:22:00"	"initialization CTD profile date/time"
+"init datetime"	"2003-10-09 11:47:00"	"initialization CTD profile date/time"
 ! End of run date/time
-"end datetime"	"2005-10-11 00:22:00"	"end of run date/time"
+"end datetime"	"2004-11-01 12:00:00"	"end of run date/time"
 "dt"		900	 "time step [s]"
 ! Implicit solver iteration limit
 "max_iter"	30	"implicit solver max iterations"
@@ -27,11 +26,11 @@
 
 ! Initial conditions
 "ctd_in"
-  "../sog-initial/ctd/SG-S3-2004-10-19.sog"
+  "../sog-initial/ctd/SG-S3-2003-10-09.sog"
   "initialization CTD file"
 ! Give a path/filename for either nuts_in or botl_in; set the other to "N/A"
 "nuts_in"
-  "../sog-initial/stratogem_nuts/Nuts-S3-2004-10-19.sog"
+  "../sog-initial/stratogem_nuts/Nuts-S3-2003-10-09.sog"
   "pre-processed nitrate & silicon profile data from STRATOGEM bottles"
 "botl_in"
   "N/A"
@@ -42,29 +41,29 @@
 
 ! Time series output files
 "std_phys_ts_out"  
-	"timeseries/std_phys_sog-test1.out"  
+	"timeseries/std_phys_R3base.out"  
 	"standard physics timeseries"
 "user_phys_ts_out" 
-	"timeseries/user_phys_sog-test1.out" 
+	"timeseries/user_phys_R3base.out" 
 	"user-defined physics timeseries"
 "std_bio_ts_out"   
-	"timeseries/std_bio_sog-test1.out"   
+	"timeseries/std_bio_R3base.out"   
 	"standard biology timeseries"
 "user_bio_ts_out"
-	"timeseries/user_bio_sog-test1.out"  
+	"timeseries/user_bio_R3base.out"  
 	"user-defined biology timeseries"
 
 ! Profiles output
 "noprof"	1		"no. of profiles to print"
 ! *** It would be nice to replace the yr-day and day-sec lists with a list
 ! *** of dates/times for profiles output
-"profday"	303		"yr-day for profile"
-"proftime"	43200.		"day-sec for profile"
-"haloclinefile"	"profiles/halo-sog-test1.out"	"file for halocline results"
-"profile_base"	"profiles/sog-test1"	"profile file base (datetime will be added)"
+"profday"	077		"yr-day for profile"
+"proftime"	48480.		"day-sec for profile"
+"haloclinefile"	"profiles/halo-R3base.out"	"file for halocline results"
+"profile_base"	"profiles/R3base"	"profile file base (datetime will be added)"
 ! Hoffmueller diagram output (a collection of profiles at time intervals
 ! for contour or colourmap plotting)
-"Hoffmueller file"	"profiles/hoff-sog-test1.dat"	"file for Hoffmueller results"
+"Hoffmueller file"	"profiles/hoff-R3base.dat"	"file for Hoffmueller results"
 "Hoffmueller start yr"	2004			"yr to start Hoff output"
 "Hoffmueller start day"	294			"yr-day to start Hoff output"
 "Hoffmueller start sec"	43200			"day-sec to start (midnight=0)"
@@ -78,10 +77,10 @@
 "temp_constant"    .false.       "Is bottom temperature constant over time?"
 ! Fit parameters for core variables
 "salinity"
-  29.62605865d0 0.10374454d0 -0.03562458d0 0.d0 -0.14156091d0 -0.06348989d0 0.d0 
+  29.62605865d0 0.10374454d0 -0.03562458d0 0.d0 -0.14156091d0 -0.06348989d0 0.d0
   "Salinity (constant, seasonal and biseasonal components)"
 "temperature"
-  9.3044d0 0.06384430d0 -0.84712324d0 0.d0 -0.05355254d0 -0.05775216d0 0.0292d0
+  9.3044d0 0.06384430d0 -0.84712324d0 0.d0 -0.05355254d0 -0.05775216d0 0.00007993d0
   "Temperature (constant, seasonal and biseasonal components)"
 "Phytoplankton"
   0.58316137d0 -0.11206845d0 0.26241523d0 0.d0 0.d0 0.d0 0.d0
@@ -93,10 +92,10 @@
   54.0d0 0.d0 0.d0 0.d0 0.d0 0.d0 0.d0
   "Silicon (constant)"
 "Ammonium"
-  1.0d0 0.d0 0.d0 0.d0 0.d0 0.d0 0.d0 
+  1.0d0 0.d0 0.d0 0.d0 0.d0 0.d0 0.d0
   "Ammonium (constant)"
 "Ratio"
-  1.25948868d0 0.d0 0.d0 0.d0 0.97697686d0 0.46289294d0 0.d0 
+  1.25948868d0 0.d0 0.d0 0.d0 0.97697686d0 0.46289294d0 0.d0
   "Ratio of pico/micro plankton (constant and biseasonal)"
 !Basin axis lengths (km)
 "Lx"            30.0d3          "Minor axis (cross-strait) of model domain [m]"
@@ -105,7 +104,7 @@
 "nu_w_m"        1.5d-4          "internal wave breaking momentum mixing"
 "nu_w_s"        1.5d-5          "internal wave breaking scalar mixing"
 "shear smooth"	0.375d0 0.5d0	"shear diffusivity smoothing parameters"
-<! Fresh water flux:
+! Fresh water flux:
 "upwell_const"  2.5d-5         "maximum upwelling velocity [m/s]"
 "Qbar"          3624d0         "mean total freshwater"
 "F_SOG"         0.12d0         "exponential for SOG component of Ft"
@@ -136,9 +135,9 @@
 
 ! Biology model parameters:
 ! Model complexity flags:
-"flagellates_on"	.true.	"include flagellates in model?"
+"flagellates_on"	.false.	"include flagellates in model?"
 "remineralization"	.true.	"include remineralization loop in model?"
-"use microzooplankton"  .true.  "include microzooplankton in model?"
+"use microzooplankton"  .false.  "include microzooplankton in model?"
 "single species light"  .false.  "high light limitation for single species case"
 ! Zooplankton values:
 ! tuned
@@ -389,7 +388,7 @@
 "Englishman integ days"  20       "Days over which to integrate Englishman River"
 
 ! Start year to run model: used in forcing.f90
-"startyear"     2004    "initialization of runtime for model"
+"startyear"     2003    "initialization of runtime for model"
 
 ! File names for forcing data (wind, met, rivers)
 "Wind"
